schema/Retail/Cancel/on_cancel_rto.ts

```diff
@@ -82,7 +82,7 @@ export const cancel_RTO_Schema = {
             id: {
               type: 'string',
               minLength: 1,
-              pattern: '^[a-zA-Z0-9]{1,32}$',
+              pattern: '^[a-zA-Z0-9-]{1,32}$|^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$',
               errorMessage: 'Should be alphanumeric upto 32 letters max',
```

Comment:
Can we specify that the error is related to order ID

Reply:
done